packages/fluent-ui/src/BaseInputTemplate/BaseInputTemplate.tsx

```diff
@@ -90,7 +91,7 @@ export default function BaseInputTemplate<
         id={id}
         name={id}
         placeholder={placeholder}
-        label={displayLabel ? label || schema.title : undefined}
+        label={labelValue(label, hideLabel)}
```

Comment:
So fluent treats empty string and undefined the same so I just dropped `schema.title`